api_tests/logs/views/test_log_added_contributors.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+import httplib as http
+
+from nose.tools import *  # noqa
+
+from tests.base import ApiTestCase
+from tests.factories import (
+    ProjectFactory,
+    AuthUserFactory
+)
+
+from framework.auth.core import Auth
+
+from website.models import NodeLog, Node
+from website.util import permissions as osf_permissions
+from api.base.settings.defaults import API_BASE
+
+
+class LogsTestCase(ApiTestCase):
+
+    def setUp(self):
+        super(LogsTestCase, self).setUp()
+
+        self.user = AuthUserFactory()
+        self.user_two = AuthUserFactory()
+
+        self.action_set = NodeLog.actions
+        self.node = ProjectFactory(is_public=False)
+
+        self.node.add_contributor(self.user, permissions=[osf_permissions.READ], auth=Auth(self.node.creator), log=True, save=True)
+        self.node_log_url = '/{}nodes/{}/logs/'.format(API_BASE, self.node._id)
+        self.url = '/{}logs/'.format(API_BASE)
+        self.log_add_contributor = self.node.logs[1]
+
+        self.public_node = ProjectFactory(is_public=True)
+        self.public_node.add_contributor(self.user, permissions=[osf_permissions.READ], auth=Auth(self.node.creator), log=True, save=True)
+
+        self.public_log_add_contributor = self.public_node.logs[1]
+
+        self.private_log_contribs_url = self.url + '{}/added_contributors/'.format(self.log_add_contributor._id)
+        self.public_log_contribs_url = self.url + '{}/added_contributors/'.format(self.public_log_add_contributor._id)
+
+    def tearDown(self):
+        NodeLog.remove()
+        Node.remove()
+
+
+class TestLogAddedContributors(LogsTestCase):
+
+    def test_log_detail_private_logged_in_contributor(self):
```

Comment:
(Though I'm not sure we need the `log_foo_` prefix since it's redundant with the name of the test case, `TestLog{Detail,AddedContributors}`.)